src/pages/elements/Report.js

```diff
@@ -381,12 +537,36 @@ const DeploymentValidations = ({ get_ns, deployment_validations}) => {
   </React.Fragment>
 }
 
-function Report({ report, namespaces}) {
+function Report({ report, namespaces, saas_files}) {
   const content = yaml.safeLoad(report.content);
 
   // fetch the namespace. Returns `undefined` if not found
   const get_ns = (c, ns) => namespaces.filter(n => n['name'] === ns && n['cluster']['name'] === c)[0];
 
+  let promotionSection;
+  if (content.production_promotions) {
```

Comment:
switch content.production_promotions to else if and add comments for removing these when we get rid of all old reports

Reply:
Got it!